hw/display/meson.build

```diff
@@ -37,6 +37,7 @@ softmmu_ss.add(when: 'CONFIG_MACFB', if_true: files('macfb.c'))
 softmmu_ss.add(when: 'CONFIG_NEXTCUBE', if_true: files('next-fb.c'))
 
 softmmu_ss.add(when: 'CONFIG_VGA', if_true: files('vga.c'))
+softmmu_ss.add(when: 'CONFIG_VIRTIO_GPU', if_true: files('virtio-dmabuf.c'))
```

Comment:
This doesn't have much relationship with virtio-gpu for now, I wonder if there is a better place to put it to build rather than depend on CONFIG_VIRTIO_GPU.

Reply:
True, for now the relationship is not clear. And even if we had added the code to GPU, probably the shared table can be argued to be device-agnostic.

Maybe `CONFIG_VIRTIO_PCI` could be a good fit?